cabal.project

```diff
@@ -0,0 +1 @@
+allow-newer: htoml-megaparsec:unordered-containers
```

Comment:
Oh, I see why the build is failing. You also need to specify `packages` in `cabal.project`

```suggestion
packages: .

allow-newer: htoml-megaparsec:unordered-containers
```

Reply:
aha